clients/src/main/java/org/apache/kafka/common/requests/ProduceResponse.java

```diff
@@ -58,7 +57,7 @@
      * {@link Errors#UNKNOWN_TOPIC_OR_PARTITION} (3)
      * {@link Errors#NOT_LEADER_FOR_PARTITION} (6)
      * {@link Errors#MESSAGE_TOO_LARGE} (10)
-     * {@link Errors#INVALID_TOPIC} (17)
+     * {@link Errors#INVALID_TOPIC_EXCEPTION} (17)
```

Comment:
Why we want to add the `_EXCEPTION` suffix? Others do not have it.

Reply:
Someone actually changed it from `INVALID_TOPIC` to `INVALID_TOPIC_EXCEPTION`: https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/protocol/Errors.java#L162

Reply:
Got it, I think it was in this case long time ago, we can fix it in another PR if we want.